netjsonconfig/backends/zerotier/converters.py

```diff
@@ -0,0 +1,17 @@
+from ..base.converter import BaseConverter
+from .schema import schema
+
+
+class ZeroTier(BaseConverter):
+    netjson_key = 'zerotier'
+    intermediate_key = 'zerotier'
+    _schema = schema
+
+    def to_intermediate_loop(self, block, result, index=None):
+        vpn = self.__intermediate_vpn(block)
+        result.setdefault('zerotier', [])
+        result['zerotier'].append(vpn)
+        return result
+
+    def __intermediate_vpn(self, config, remove=[False, 0, '']):
```

Comment:
never use a mutable structure as default argument, read more about this here: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/

Reply:
noted!